### ISSUE TITLE
[FUNCTIONALITY]: Sortable Table - SHOULD have a base Table component with proper HTML structure

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9197]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [sortable table component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/sortabletable/) would benefit greatly from a basic `<Table />` component. This way we could build out more complex tables as extensions than completely new components. Thinking:

* Tables with multiple heading rows
* Tables with rows or columns that span multiple
* Tables with sortable columns

## Related Issues
* #9193 
* #9194 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria
- [ ] HTML validates with W3C validator or other HTML5 checker
- [ ] No axe errors
- [ ] Base component is used to create extended or enhanced table components
- [ ] Visual styling does not change, except where noted by issue tickets

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/sortabletable/

## Possible Fixes (optional)
This example uses proper `scope` attributes and a `<th>` for first table row element. This is the preferred means to create table structure, and is well-supported by assistive technology.

```diff
<table>
  <thead>
    <tr>
+     <th scope="col">Name</th>
+     <th scope="col">Email</th>
+     <th scope="col">Message</th>
    </tr>
  </thead>
  <tbody>
    <tr>
+     <th scope="row">Joe</th>
      <td>joe@joe.com</td>
      <td>Hello</td>
    </tr>
    <tr>
+     <th scope="row">Bob</th>
      <td>bob@bob.com</td>
      <td>Hola</td>
    </tr>
    <tr>
+     <th scope="row">Tom</th>
      <td>tom@tom.com</td>
      <td>What's Up</td>
    </tr>
  </tbody>
</table>
```

## Screenshots or Trace Logs
![Screen Shot 2020-05-18 at 10 39 35 AM](https://user-images.githubusercontent.com/934879/82232397-32cd9f00-98f4-11ea-8ab3-91c24eaa434e.png)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #229](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/229) via [**ZenHub**](https://www.zenhub.com/)
